deploy.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# TODO: make this with tempfile
+KEYFILE=__TEMP_INPUT_KEY_FILE
+
+echo "${WWW_HASKELL_ORG_SSH_KEY}" > "$KEYFILE"
+chmod 600 "$KEYFILE"
+
+# WARNING: --delete is dangerous.  Be absolutely sure you are
```

Comment:
Would it make sense to do a dry run by default, and have a special `--run` flag in this script to do the actual `--delete`? The CI script could pass the flag in, but this way the default isn't destructive if anyone is testing things manually.

Reply:
I think Davean will set us up a `www-test` directory that will be deployed to https://www-test.haskell.org/. It would be good to have a flag on this script which chooses between live and test. I will confirm with Davean.

Reply:
OK, looks like this is in the pipeline so I added that flag. Not yet enabled.

Reply:
Now enabled